package.json

```diff
@@ -6,11 +6,15 @@
   "scripts": {
     "lint": "biome check --write .",
     "lint:ci": "biome ci",
-    "changeset": "changeset"
+    "changeset": "changeset",
+    "release": "changeset publish",
+    "packages:build": "pnpm -r --filter=\"@ensnode/*\" --filter=\"!@ensnode/shared-configs\" build",
+    "packages:dev": "pnpm -r --filter=\"@ensnode/*\" --filter=\"!@ensnode/shared-configs\" dev"
```

Comment:
We might not need those two steps, if we decide to follow [this suggestion](https://github.com/namehash/ensnode/pull/262/files#r1958161411).